scss/components/_tags-modal.scss

```diff
@@ -0,0 +1,33 @@
+.___outside-namespace {
+  .civicase__tags-modal {
+    background-color: $crm-white !important;
+  }
+}
+
+.civicase__tags-modal__generic-tags__container {
+  border: 1px solid $crm-grayblue-dark;
+  border-radius: 2px;
+  padding: 5px;
+
+  input {
+    margin-top: 0 !important;
+  }
+
+  label {
+    margin-bottom: 0;
+  }
+}
+
+.civicase__tags-modal__tagsets__container {
+  label {
+    margin-top: 4px;
+  }
+
+  .col-sm-9 {
```

Comment:
Why do we need to set the width to 75% explicitly. Won't `9 col out of 12` will have 75% width by default?